sources/moves.py

```diff
@@ -182,4 +202,12 @@ def create_datafiles(access_token, task_id=None, update_url=None, **kwargs):
         print 'Please specify a token and directory.'
         sys.exit(1)
 
-    create_datafiles(*sys.argv[1:-1], filedir=sys.argv[-1])
+    while True:
+        result = create_datafiles(*sys.argv[1:-1], filedir=sys.argv[-1])
+        if result:
+            countdown = result['countdown']
```

Comment:
will this fail if you run it on a user with data that's not larger enough to cause requeueing? (since 'countdown' won't be in `result`?)

or in that case will result just be `None`?